conans/model/workspace.py

```diff
@@ -116,12 +121,18 @@ def _loads(self, text):
         if not self._root:
             raise ConanException("Conan workspace needs at least 1 root conanfile")
 
-        editables = yml.pop("editables", None)
+        editables = yml.pop("editables", {})
         for package_name, data in editables.items():
+            if not data:
```

Comment:
Shouldn't the validation go to the `LocalPackage`? Which fields are mandatory? Only the `path`?

Reply:
Yes, only the path is really mandatory

Reply:
Then it should complain because the path is missing, the message about the missing data is a bit weird.

Reply:
Done